orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml

```diff
@@ -0,0 +1,218 @@
+version: 2
+models:
+  - name: mrt_global__booking
+    columns:
+      - name: booking_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
+          - unique
+      - name: booking_creation_date
+        description: ''
+        data_type: DATE
+      - name: booking_created_at
+        description: ''
+        data_type: DATETIME
+      - name: booking_quantity
+        description: ''
+        data_type: INT64
+      - name: booking_amount
+        description: ''
+        data_type: NUMERIC
+      - name: booking_status
+        description: ''
+        data_type: STRING
+      - name: booking_is_cancelled
+        description: ''
+        data_type: BOOLEAN
+      - name: booking_is_used
+        description: ''
+        data_type: BOOLEAN
+      - name: booking_cancellation_date
+        description: ''
+        data_type: DATETIME
+      - name: booking_cancellation_reason
+        description: ''
+        data_type: STRING
+      - name: user_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
+      - name: deposit_id
+        description: ''
+        data_type: 
+      - name: deposit_type
+        description: ''
+        data_type: STRING
+      - name: reimbursed
+        description: ''
+        data_type: BOOLEAN
+      - name: booking_intermediary_amount
+        description: ''
+        data_type: NUMERIC
+      - name: booking_rank
+        description: ''
+        data_type: INT64
+      - name: booking_used_date
+        description: ''
+        data_type: DATETIME
+      - name: stock_beginning_date
+        description: ''
+        data_type: DATETIME
+      - name: stock_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
+      - name: offer_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
```

Comment:
Pareil ici on a un filtre dans le modèle `AND s.offer_id IS NOT NULL` donc pas sûr que ce test soit nécessaire